src/Http/Authentication.Core/src/AuthenticationSchemeProvider.cs

```diff
@@ -184,15 +184,14 @@ public virtual void AddScheme(AuthenticationScheme scheme)
     /// <param name="name">The name of the authenticationScheme being removed.</param>
     public virtual void RemoveScheme(string name)
     {
-        if (!_schemes.ContainsKey(name))
+        if (!_schemes.TryGetValue(name, out var scheme))
```

Comment:
There is also `GetSchemeAsync` which can be updated, but that'll probably show up in the other PR.

Reply:
Should this one be changed given the out value is discarded?
Or:
```suggestion
        if (!_schemes.TryGetValue(name, out var _))
```

Reply:
Looks like #44799 found that and 4 others. Rather than churn this PR, shall I just change this and address those with a rebase once this is merged?

Reply:
Whatever's convenient for you.